tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/SeqModelCheckerListener.scala

```diff
@@ -0,0 +1,31 @@
+package at.forsyte.apalache.tla.bmcmt
+
+import at.forsyte.apalache.tla.bmcmt.trex.DecodedExecution
+import at.forsyte.apalache.tla.lir.{TlaEx, TlaModule}
+
+/**
+ * Observe [[SeqModelChecker]]. State changes in model checker state are reported via callbacks.
+ */
+trait SeqModelCheckerListener {
```

Comment:
```suggestion
trait ModelCheckerListener {
```

It's not really related to Sequential vs. Parallel. (The parallel version is living in a branch, but it very unlikely that we will merge it.)

Reply:
Good point!